### DISCUSSION TITLE
Batch of helpful print models for the AD5M, designed or remixed by Doug Joseph - Design8Studio

### DISCUSSION THREAD

**Discussion by DougJoseph** [xblax/flashforge_ad5m_klipper_mod#87] [Show and tell]
**First is the "Flashforge Adventurer 5M (AD5M) - simpler, easier, faster drag chain termination at hotend (v1.0)"
This short video explains everything.**

https://www.youtube.com/watch?v=A9MNIt4pA_Y

Printables link: https://www.printables.com/model/785667-flashforge-adventurer-5m-ad5m-simpler-easier-faste

![IMG_7335](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/40018d17-cd02-4366-b08d-b2d1715f41d6)

**Second is "Flashforge Adventurer 5M (AD5M) Printed Camera Mounts (two to choose from) v1.0"**

Printables link: https://www.printables.com/model/813334-flashforge-adventurer-5m-ad5m-printed-camera-mount

Flashforge's USB camera is stock on AD5M Pro. For the Adventurer 5M (non Pro model) it can be ordered factory direct from Flashforge's site. It mounts just inside the right front upright, as shown. However, if you have installed the DIY printed enclosure touted by Flashforge, **then the cable on their factory camera is too short**. My printed extended camera mount positions the camera higher, and makes use of the short cable doable.

![AD5M DIY Printed Camera Mount (04) comparison front (IMG_7433)](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/a9c24176-abcc-47f7-bc94-0d3ac2ef081f)

![AD5M DIY Printed Camera Mount (05) comparison back (IMG_7434)](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/fe0747ad-7e01-4d26-890a-0080f60680ea)

![AD5M DIY Printed Camera Mount (01) with text (IMG_7436)](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/4b08fbe9-f34f-4443-b5a6-0a2c00e5bb6c)

![AD5M DIY Printed Camera Mount (03) highlighted (IMG_7437)](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/1456bfbf-4475-4c58-842f-5434b43a9849)

![AD5M Camera Module PCB 12x30mm](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/58c8b01b-2785-42da-98bc-fbbae484a545)

![AD5M Camera mount, bottom view (factory part, injection molded plastic)](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/643c7643-38c2-4334-b565-a0e71f38edff)

![AD5M Camera mount, right side view 1 (factory part, injection molded plastic)](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/0f72004c-2bc9-4a68-8024-66fad5f7a1f2)

![AD5M Camera mount, right side view 2 (factory part, injection molded plastic)](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/a2df5267-56e6-40d4-a8da-3d4296396a7c)

![AD5M Camera mount, top view (factory part, injection molded plastic)](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/b4126a6f-0bda-4d83-872f-fc726924b6f6)

**Third - newly added - Flashforge Adventurer 5M (AD5M) Nozzle Storage Box - two to choose from, including one that attaches to back of printer v1.0**

Printable files: https://www.printables.com/model/814546-flashforge-adventurer-5m-ad5m-nozzle-storage-box-t

This short video explains everything:

https://www.youtube.com/watch?v=D21v0OObVsc

Cool storage for four nozzles for Flashforge Adventurer 5 (AD5M).

Uses 1.75mm filament as the hinge pin.

Two options to choose from:

* Box and lid
* Box and lid that mount onto back of AD5M printer, whenever the DIY printed enclosure is in use. The enclosure mounts the spool holder in a different spot, which frees up the two screw holes that formerly attached it. The mount in this design is intended to attach by using those same screw holes. Attach the mount with two screws, then slide the box base onto the mount.

**Print settings**
Print as oriented. Prints without need of supports. I used 10% infill on the lid, and 15% infill on the base and the mount. I used 2 perimeter walls, with a 0.4 mm nozzle.

My PayPal tip jar: https://paypal.me/design8studio

![2024-03-22 Flashforge Adventurer 5 (AD5M) Nozzle Storage Box with option to attach to back of printer - Video-Thumbnail](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/56c61c22-66ef-4bfa-bb6c-ce17b06ef7dc)

![IMG_7444 screen grab 01 COVER](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/dfa30b4f-a5aa-4dcc-9377-260a85eb3ef1)

![IMG_7445](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/eaa4d74b-580e-43a6-812f-9a0c265853be)

![IMG_7446](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/5c04e727-6df9-4ab4-b07b-c15b79f501b6)

![IMG_7444 screen grab 02](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/bf950302-25e8-4cf3-bafa-12864d729b61)

**Fourth: Flashforge Adventurer 5 (AD5M) LED mod accessory - buck converter MOUNT for inside back of printer (24v -> 12v inline, etc)**

Printables link: https://www.printables.com/model/827706-flashforge-adventurer-5-ad5m-led-mod-accessory-buc

![IMG_7538](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/b9f5e1f1-73ca-48dd-8ba7-af438905324d)

For adding LED lighting to the non-pro version of the Flashforge Adventurer 5 (AD5M), you must either buy 24v LEDs, or use a super affordable buck converter to knock the voltage down to whatever your LEDs on hand need. I had gobs of 12v LEDs in stock from previous projects. This LED mod accessory makes it easy to mount the buck converter inside the back of the printer, just above the main board, by using the screw holes on the upper of the two unused air filter fan tunnels.

Printed mount for affordable buck converter to use 12v LED's in your AD5M (or 5v LEDs or whatever). 

Prints as oriented. Prints without any need of supports.

This mod leaves the air filter fan tunnel open, and it's possible that a mod to add a filter and fan could be used in conjunction with this mod (no guarantees). 

Has 3mm stand-offs on the mount, to give a little air behind the buck converter's PCB to hopefully help with heat dissipation.

NOTE: this was made for these buck converters (affiliate link - adds nothing to your price, and may give a small percent to support my channel):

https://amzn.to/3TDBBB5

[6 Pack LM2596 DC to DC High Efficiency Voltage Regulator 3.2-35V to 1.25-30V Buck Converter DIY Power Supply Step-Down Module](https://amzn.to/3TDBBB5)

![71lLCBgYohL _SL1500_](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/7b9af6c6-2066-4753-a25d-38f9e553d2d5)

See pics for where it's mounted and how it's oriented.

![IMG_7539](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/8bac3290-44d7-4d4d-91b6-07aa17e98629)

<img width="1366" alt="Screenshot 2024-03-31 at 8 12 02 PM" src="https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/8bd3f239-3c5f-4ab5-a88b-f7df54e40212">

**Fifth: Flashforge Adventurer 5 (AD5M) LED Light Bars mod accessories - mounts for 5050 LED (2 strips per mount bar) v1.0**

Printables link: https://www.printables.com/model/828632-flashforge-adventurer-5-ad5m-led-light-bars-mod-ac

![IMG_7547](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/096b5c08-9d6b-4878-8d1a-8a1c9971a30d)
![IMG_7548](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/5491f618-3638-4499-b266-b5fc1d26816d)
![IMG_7549](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/39ffca2a-5c14-4e06-b421-ddf05077befb)

NOTE: an explainer / how-to video is coming soon.

These are printed accessories for adding LED light bars to the Flashforge Adventurer 5 (AD5M) printer.

**Printing info:**
- Print as oriented
- No need of supports.

**Installation info:** 
Use these printed mount to add two strips (per mount) of 5050 LEDs to the front and both sides. I recommend use of non-waterproof LEDs, or removing the waterproofing layer from the strips, especially on the front. The space allowance on the front is very tight. The waterproofing layer can be OK on the sides. I've provided two variants of the “LED Strip Clasps” for the sides: one is for LEDs with the waterproofing layer still in place, while the other is for non-waterproof LEDs, or for waterproof LEDs that have had that layer peeled off.

The side mount strips have sockets for magnets. They can be attached using magnets, or if you could choose to use double stick  tape. The front needs to be attached with double stick tape, because there is not enough space for magnets in the printer regarding the front. If using magnets on the side mounts, CA glue in 16 round 2 mm x 5 mm magnets per mount.

Note the orientation for use is not the same as for printing. For use, wider part faces to the top, and narrower part faces to the side.

![00 Screenshot 2024-04-01 at 2 00 54 PM](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/2f49e20e-9282-4585-9dc2-f5dd5ad97b9f)
![01 Screenshot 2024-04-01 at 1 31 22 PM](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/6d247622-0d0d-40d5-ac84-418a142f6126)
![02 Screenshot 2024-04-01 at 1 32 00 PM](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/6e51d37f-8f56-428a-a12d-c342589823fb)
![03 Screenshot 2024-04-01 at 1 33 20 PM](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/1df12dc9-c751-4800-9d4f-f3fb221bafc3)
![04 Screenshot 2024-04-01 at 1 33 04 PM](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/b5a36c7f-1e0c-42d0-bbe5-d24f45aebe1c)
![05 Screenshot 2024-04-01 at 1 33 56 PM](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/43d38580-a9d9-40d6-b991-73ac8c1f60bc)
![06 Screenshot 2024-04-01 at 1 34 17 PM](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/61f333f9-1f5d-41f4-b906-cb80a7eff76c)
![07 Screenshot 2024-04-01 at 1 34 50 PM](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/2f5eacfa-e795-4935-ac5f-a84ecc37b338)
![08 Screenshot 2024-04-01 at 1 35 03 PM](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/6bfbe4be-931d-4fad-8903-4e71e631f0e6)
![09 Screenshot 2024-04-01 at 1 30 07 PM](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/c7fafece-93f6-423d-95e3-081fe7b8fa6b)

[![2024-04-06 Flashforge Adventurer 5M (AD5M) IMPROVED Spool Holder + SIDE MOUNT Spool Holder - Video-Thumbnail](https://github.com/xblax/flashforge_ad5m_klipper_mod/assets/8921397/90a417bb-701f-4e36-8928-d519bfc2aee6)](https://www.printables.com/model/886714-flashforge-adventurer-5m-ad5m-filament-cutter-comp)

**Other cool options for the AD5M**
 
Summary: Printables links to my AD5M-related design work:
https://www.printables.com/model/785667-flashforge-adventurer-5m-ad5m-simpler-easier-faste
https://www.printables.com/model/813334-flashforge-adventurer-5m-ad5m-printed-camera-mount 
https://www.printables.com/model/814546-flashforge-adventurer-5m-ad5m-nozzle-storage-box-t
https://www.printables.com/model/827706-flashforge-adventurer-5-ad5m-led-mod-accessory-buc 
https://www.printables.com/model/828632-flashforge-adventurer-5-ad5m-led-light-bars-mod-ac
https://www.printables.com/model/886714-flashforge-adventurer-5m-ad5m-filament-cutter-comp

My PayPal tip jar: https://paypal.me/design8studio

**Comment by xblax** [comment]
Thanks for sharing, maybe I will the camera mount a try!

**Comment by DougJoseph** [comment]
I created another remix for a printable enhancement of the AD5M:

Printable files: https://www.printables.com/model/814546-flashforge-adventurer-5m-ad5m-nozzle-storage-box-t

This short video explains everything:

https://www.youtube.com/watch?v=D21v0OObVsc

Cool storage for four nozzles for Flashforge Adventurer 5 (AD5M).

Uses 1.75mm filament as the hinge pin.

Two options to choose from:

* Box and lid
* Box and lid that mount onto back of AD5M printer, whenever the DIY printed enclosure is in use. The enclosure mounts the spool holder in a different spot, which frees up the two screw holes that formerly attached it. The mount in this design is intended to attach by using those same screw holes. Attach the mount with two screws, then slide the box base onto the mount.

**Print settings**
Print as oriented. Prints without need of supports. I used 10% infill on the lid, and 15% infill on the base and the mount. I used 2 perimeter walls, with a 0.4 mm nozzle.

My PayPal tip jar: https://paypal.me/design8studio

![2024-03-22 Flashforge Adventurer 5 (AD5M) Nozzle Storage Box with option to attach to back of printer - Video-Thumbnail](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/56c61c22-66ef-4bfa-bb6c-ce17b06ef7dc)

![IMG_7444 screen grab 01 COVER](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/dfa30b4f-a5aa-4dcc-9377-260a85eb3ef1)

![IMG_7445](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/eaa4d74b-580e-43a6-812f-9a0c265853be)

![IMG_7446](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/5c04e727-6df9-4ab4-b07b-c15b79f501b6)

![IMG_7444 screen grab 02](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/bf950302-25e8-4cf3-bafa-12864d729b61)

**Reply by consp** [reply]
Looks nice! When I reinstall the backplate I'm definitely going to print one.

**Comment by DougJoseph** [comment]
I created another remix for a printable enhancement of the AD5M (fourth one so far):

**Flashforge Adventurer 5 (AD5M) LED mod accessory - buck converter MOUNT for inside back of printer (24v -> 12v inline, etc)**

Printables link: https://www.printables.com/model/827706-flashforge-adventurer-5-ad5m-led-mod-accessory-buc

![IMG_7538](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/b9f5e1f1-73ca-48dd-8ba7-af438905324d)

For adding LED lighting to the non-pro version of the Flashforge Adventurer 5 (AD5M), you must either buy 24v LEDs, or use a super affordable buck converter to knock the voltage down to whatever your LEDs on hand need. I had gobs of 12v LEDs in stock from previous projects. This LED mod accessory makes it easy to mount the buck converter inside the back of the printer, just above the main board, by using the screw holes on the upper of the two unused air filter fan tunnels.

Printed mount for affordable buck converter to use 12v LED's in your AD5M (or 5v LEDs or whatever). 

Prints as oriented. Prints without any need of supports.

This mod leaves the air filter fan tunnel open, and it's possible that a mod to add a filter and fan could be used in conjunction with this mod (no guarantees). 

Has 3mm stand-offs on the mount, to give a little air behind the buck converter's PCB to hopefully help with heat dissipation.

NOTE: this was made for these buck converters (affiliate link - adds nothing to your price, and may give a small percent to support my channel):

https://amzn.to/3TDBBB5

[6 Pack LM2596 DC to DC High Efficiency Voltage Regulator 3.2-35V to 1.25-30V Buck Converter DIY Power Supply Step-Down Module](https://amzn.to/3TDBBB5)

![71lLCBgYohL _SL1500_](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/7b9af6c6-2066-4753-a25d-38f9e553d2d5)

See pics for where it's mounted and how it's oriented.

![IMG_7539](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/8bac3290-44d7-4d4d-91b6-07aa17e98629)

<img width="1366" alt="Screenshot 2024-03-31 at 8 12 02 PM" src="https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/8bd3f239-3c5f-4ab5-a88b-f7df54e40212">
 
**Other cool options for the AD5M**
 
You may also like my other AD5M related design work:
https://www.printables.com/model/785667-flashforge-adventurer-5m-ad5m-simpler-easier-faste
https://www.printables.com/model/813334-flashforge-adventurer-5m-ad5m-printed-camera-mount 
https://www.printables.com/model/814546-flashforge-adventurer-5m-ad5m-nozzle-storage-box-t 

My PayPal tip jar: https://paypal.me/design8studio

**Comment by DougJoseph** [comment]
I created another remix for a printable enhancement of the AD5M (fifth one so far):

**Flashforge Adventurer 5 (AD5M) LED Light Bars mod accessories - mounts for 5050 LED (2 strips per mount bar) v1.0**
Printables link: https://www.printables.com/model/828632-flashforge-adventurer-5-ad5m-led-light-bars-mod-ac

![IMG_7547](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/096b5c08-9d6b-4878-8d1a-8a1c9971a30d)
![IMG_7548](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/5491f618-3638-4499-b266-b5fc1d26816d)
![IMG_7549](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/39ffca2a-5c14-4e06-b421-ddf05077befb)

NOTE: an explainer / how-to video is coming soon.

These are printed accessories for adding LED light bars to the Flashforge Adventurer 5 (AD5M) printer.

**Printing info:**
- Print as oriented
- No need of supports.

**Installation info:** 
Use these printed mount to add two strips (per mount) of 5050 LEDs to the front and both sides. I recommend use of non-waterproof LEDs, or removing the waterproofing layer from the strips, especially on the front. The space allowance on the front is very tight. The waterproofing layer can be OK on the sides. I've provided two variants of the “LED Strip Clasps” for the sides: one is for LEDs with the waterproofing layer still in place, while the other is for non-waterproof LEDs, or for waterproof LEDs that have had that layer peeled off.

The side mount strips have sockets for magnets. They can be attached using magnets, or if you could choose to use double stick  tape. The front needs to be attached with double stick tape, because there is not enough space for magnets in the printer regarding the front. If using magnets on the side mounts, CA glue in 16 round 2 mm x 5 mm magnets per mount.

Note the orientation for use is not the same as for printing. For use, wider part faces to the top, and narrower part faces to the side.

![00 Screenshot 2024-04-01 at 2 00 54 PM](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/2f49e20e-9282-4585-9dc2-f5dd5ad97b9f)
![01 Screenshot 2024-04-01 at 1 31 22 PM](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/6d247622-0d0d-40d5-ac84-418a142f6126)
![02 Screenshot 2024-04-01 at 1 32 00 PM](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/6e51d37f-8f56-428a-a12d-c342589823fb)
![03 Screenshot 2024-04-01 at 1 33 20 PM](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/1df12dc9-c751-4800-9d4f-f3fb221bafc3)
![04 Screenshot 2024-04-01 at 1 33 04 PM](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/b5a36c7f-1e0c-42d0-bbe5-d24f45aebe1c)
![05 Screenshot 2024-04-01 at 1 33 56 PM](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/43d38580-a9d9-40d6-b991-73ac8c1f60bc)
![06 Screenshot 2024-04-01 at 1 34 17 PM](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/61f333f9-1f5d-41f4-b906-cb80a7eff76c)
![07 Screenshot 2024-04-01 at 1 34 50 PM](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/2f5eacfa-e795-4935-ac5f-a84ecc37b338)
![08 Screenshot 2024-04-01 at 1 35 03 PM](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/6bfbe4be-931d-4fad-8903-4e71e631f0e6)
![09 Screenshot 2024-04-01 at 1 30 07 PM](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/c7fafece-93f6-423d-95e3-081fe7b8fa6b)



**Other cool options for the AD5M**
 
Summary: Printables links to my AD5M-related design work:
https://www.printables.com/model/785667-flashforge-adventurer-5m-ad5m-simpler-easier-faste
https://www.printables.com/model/813334-flashforge-adventurer-5m-ad5m-printed-camera-mount 
https://www.printables.com/model/814546-flashforge-adventurer-5m-ad5m-nozzle-storage-box-t
https://www.printables.com/model/827706-flashforge-adventurer-5-ad5m-led-mod-accessory-buc 
https://www.printables.com/model/828632-flashforge-adventurer-5-ad5m-led-light-bars-mod-ac

My PayPal tip jar: https://paypal.me/design8studio

**Comment by DougJoseph** [comment]
# Yet another great improvement to an already great printer: 

[![2024-04-06 Flashforge Adventurer 5M (AD5M) IMPROVED Spool Holder + SIDE MOUNT Spool Holder - Video-Thumbnail](https://github.com/xblax/flashforge_ad5m_klipper_mod/assets/8921397/90a417bb-701f-4e36-8928-d519bfc2aee6)](https://www.printables.com/model/886714-flashforge-adventurer-5m-ad5m-filament-cutter-comp)

Printables link:
https://www.printables.com/model/886714-flashforge-adventurer-5m-ad5m-filament-cutter-comp


**Assembly Video**
This assembly video is good. I did not make it, but it's quite valid for my remix too. It's one that mine is based on:
https://www.youtube.com/watch?v=rSI0_ej1MC8

**Installation Video**
https://www.youtube.com/watch?v=fyWoScZqBKs

## Instructions for assembly and installation 

**Assembly**

1. Print both parts (cutter body and cutter button). If you are unsure which button to print, go with the medium one, marked 0.7 slot.
2. Take one piece of snap-off blade and put it into upper part. Use some pliers to avoid an injury. There is a small mark near blade pocket which shows you where the tip of the blade should be. Additionally you can use super glue to fix it in position.
3. Put both screws into the holes.
4. Put ball point pen springs into the holes.
5. Place the button on the main cutter body and press it fully.
6. Tighten the screws. Be careful not to over tighten.
7. Release upper part and check if blade is not visible inside the filament hole. If it is, then loosen the screws a little bit.
8. It is recommended to put some lubricant inside to keep the mechanism running smoothly. Here's an [affiliate link to some lithium grease](https://amzn.to/4dR7i3s), which is a good choice.

**Installation without DIY enclosure** 

1. Withdraw the PTFE tube from the AD5M extruder / hotend assembly.
2. Cut the existing filament using your side cutters that came with the AD5M.
3. Insert the assembled filament cutter, with the button facing to the left.
4. Insert the PTFE tube into top of the new cutter. 
5. If the PTFE tube is now too long, cut some off, and then reinsert.

**Installation with DIY enclosure, assuming you are using  my drag chain termination part [available here](https://www.printables.com/model/785667-flashforge-adventurer-5m-ad5m-simpler-easier-faste)**

1. Loosen and remove the two screws that hold the drag chain termination in place. Save the screws for re-use. Temporarily place the drag chain termination to the side.
2. Withdraw the PTFE tube from the AD5M extruder / hotend assembly.
3. Cut the existing filament using your side cutters that came with the AD5M.
4. Insert the assembled filament cutter, with the button facing to the left.
5. Insert the PTFE tube into top of the new cutter. 
6. If the PTFE tube is now too long, cut some off, and then reinsert.
7. Reinstall the drag chain termination using the two screws you saved.

**Other cool options for the AD5M**
 
You may also like my other AD5M related design work:
https://www.printables.com/model/785667-flashforge-adventurer-5m-ad5m-simpler-easier-faste
https://www.printables.com/model/813334-flashforge-adventurer-5m-ad5m-printed-camera-mount 
https://www.printables.com/model/814546-flashforge-adventurer-5m-ad5m-nozzle-storage-box-t 
https://www.printables.com/model/827706-flashforge-adventurer-5-ad5m-led-mod-accessory-buc
https://www.printables.com/model/828632-flashforge-adventurer-5-ad5m-led-light-bars-mod-ac
https://www.printables.com/model/834011-flashforge-adventurer-5m-ad5m-side-cutter-mount-v1 
https://www.printables.com/model/834097-flashforge-adventurer-5m-ad5m-improved-spool-holde 
https://www.printables.com/model/886714-flashforge-adventurer-5m-ad5m-filament-cutter-comp  

My PayPal tip jar: https://paypal.me/design8studio
